docs/proposals/20200220-cluster-resource-set.md

```diff
@@ -94,105 +96,167 @@ None. We are planning to implement this feature without modifying any of the exi
 
 #### ClusterResourceSet Object Definition
 
-This is the CRD that has a set of components to be applied to clusters that match the label selector in it.
+This is the CRD that has a set of components (resources) to be applied to clusters that match the label selector in it.
 
-The resources field is a list of secrets/configmaps in the same namespace. The clusterSelector field is a Kubernetes [label selector](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#resources-that-support-set-based-requirements) that matches against labels on clusters (only the clusters in the same namespace with the ClusterResourceSet resource).
+The resources field is a list of `Secrets`/`ConfigMaps` in the same namespace. The clusterSelector field is a Kubernetes [label selector](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#resources-that-support-set-based-requirements) that matches against labels on clusters (only the clusters in the same namespace with the ClusterResourceSet resource).
 ClusterResourceSet is namespace-scoped, all resources and clusters needs to be in the same namespace as the ClusterResourceSet.
 *Sample ClusterResourceSet YAML*
 
 ```yaml
 ---
-apiVersion: cluster.x-k8s.io/v1alpha3
+apiVersion: addons.cluster.x-k8s.io/v1alpha3
 kind: ClusterResourceSet
 metadata:
- name: crs-conf
+ name: crs1
  namespace: default
 spec:
  mode: "ApplyOnce"
  clusterSelector:
    matchLabels:
-     postcreatelabelcni: calico
+     cni: calico
  resources:
-   - name: calico-addon
+   - name: db-secret
      kind: Secret
-   - name: network-policy-addon
+   - name: calico-addon
      kind: ConfigMap
-status:
- LastUpdated:  "2020-05-05T08:24:17Z"
 ```
 
 Initially, the only supported mode will be `ApplyOnce` and it will be the default mode if no mode is provided. In the future, we may consider adding a `Reconcile` mode that reapplies the resources on resource hash change and/or periodically.
 If ClusterResourceSet resources will be managed by an operator after they are applied by ClusterResourceSet controller, "ApplyOnce" mode must be used so that reconciliation on those resources can be delegated to the operator.
 
-Each item in the resources specifies a kind (must be either ConfigMap or Secret) and a name. Each referenced ConfigMap/Secret  contains yaml/json content as value.
+Each item in the resources specifies a kind (must be either ConfigMap or Secret) and a name. Each referenced ConfigMap/Secret contains yaml/json content as value.
```

Comment:
Might be worth to specify those resources should be in the same namespace of the CRS